_includes/master/kubernetes/k8s_binary_init.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
actually given that the file list may be passed in too, I think it would need to be:

`calico-k8s-bineries [--download] [--install] files...`